src/containers/treePage/index.tsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import PageLayout from '../../components/pageLayout';
+import ReturnButton from '../../components/returnButton';
+import { Row, Col, Image, Typography, Button, Card } from 'antd';
+import { Routes } from '../../App';
+import { Helmet } from 'react-helmet';
+import { SiteProps, SiteEntry, TreeCare, SiteEntryNames } from './ducks/types';
+import PageHeader from '../../components/pageHeader';
+import { TitleProps } from 'antd/lib/typography/Title';
+import StewardshipForm from '../../components/stewardshipForm';
+import placeholder from '../../assets/images/placeholder.png';
+import { connect } from 'react-redux';
+import { LIGHT_GREY, DARK_GREEN, TEXT_GREY } from '../../utils/colors';
+import { Gap } from '../../components/themedComponents';
+import styled from 'styled-components';
+
+const { Paragraph, Title } = Typography;
+
+const StyledCard = styled(Card)`
+  width: 250px;
+  height: 100px;
+  border: solid 1px ${LIGHT_GREY};
+  margin: 7px;
+`;
+
+const TreePageContainer = styled.div`
+  width: 95vw;
+  margin: 30px auto auto;
+`;
+
+const TreeMainContainer = styled.div`
+  margin: 50px 30px 30px;
+`;
+
+const TreeImage = styled(Image)`
+  margin-top: 20px;
+`;
+
+const TreeInfoContainer = styled.div`
+  width: 90%;
+  margin: auto;
+`;
+
+const EntrySpace = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const TreeCareTitle = styled(Paragraph)`
+  margin: 15px 15px 50px;
+  font-size: 24px;
+  font-weight: bold;
+  color: ${ DARK_GREEN }
+`;
+
+const TreeCareContainer = styled.div`
+  border: solid 1px ${LIGHT_GREY};
+  height: 80%;
+  margin-top: 80px;
+  overflow: auto;
+`;
+
+const CareEntry = styled.div`
+  margin: 15px;
+`;
+
+const StewardshipContainer = styled.div`
+  margin-top: 40px;
+`;
+
+const EntryDate = styled(Paragraph)<TitleProps>`
+  display: inline;
+  text-align: center;
+  line-height: 0px;
+  font-size: 18px;
+  font-weight: bold;
+  color: ${ DARK_GREEN }
+`;
+
+const EntryMessage = styled(Paragraph)`
+  display: inline;
+  text-align: center;
+  line-height: 0px;
+  color: ${TEXT_GREY};
+`;
+
+const dummyCare: TreeCare[] = [
```

Comment:
❓ Do you want to keep dummy data for now?

Reply:
'd like to keep it for now in case anyone wants to interact with the page. I'll remove it when connecting to the backend